### PR TITLE
Add @emoji support.

### PR DESCRIPTION
**Pull request by rjchee**
Does @emoji summon whoever emoji is?

### CODE REVIEW

**Comment by rjchee** [comment]
Also added documentation and stuff.

File: .gitignore [hunk 1]
```diff
@@ -17,4 +17,4 @@ temp_pokemon.png
 
 # Secret files
 secret.txt
-
+*.swp
```
Comment (renxinhe) [inline .gitignore]: What is this for?

**Comment by renxinhe** [comment]
@rjchee can you also bump the version to `0.2.4` in `package.json`, and add `0.2.4: Added set emoji functionality.` to `README.md`?

**Comment by rjchee** [comment]
Lol I added it because I wanted to ignore my vim swap files. Let me know if
you want that removed. Will bump version when I go home.

**Pull request merged**